src/lasp_process.erl

```diff
@@ -76,6 +77,40 @@ start_tracked_process(EventCount, [ReadFuns, TransFun, {To, _}=WriteFun]) ->
         end
     end.
 
+%% @doc Track a function in the dag.
+%%
+%%      Given the input and output variables, and a function,
+%%      create a single-fire lasp process representing the dataflow
+%%      computation.
+%%
+%%      This function is synchronous, as it waits for a value to
+%%      be returned.
+%%
+single_fire_function(From, To, Fn, Args) ->
+    Self = self(),
+    %% Unique reference, ignore any other messages.
+    %% @todo Remove?
```

Comment:
I wouldn't remove, the reference seems useful here for matching only the message that you are expecting.